src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static string ToDisplayString(this ImmutableArray<SymbolDisplayPart> part
                 var actualBuilder = pool.Builder;
                 foreach (var part in parts)
                 {
-                    actualBuilder.Append(part);
+                    actualBuilder.Append(part.ToString());
```

Comment:
hrmm... what does this do?

Reply:
`part` is a value type which was being passed to `Append(object)`. Now it calls `part.ToString()` and passes it to `Append(string)`.

Reply:
that is truly truly unfortunate. :)

Reply:
@Youssef1313 may also address the issue you were having.

Reply:
➡️ Added.